src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Comment/Comment.ts

```diff
@@ -357,75 +353,6 @@ export var adhCommentListing = (
- * poolPath and key. ExternalResource is a commentable.
- *
- * If not, it is created on the fly.
- *
- * If it exists, the corresponding comment listing is created.
- */
-export var adhCreateOrShowCommentListing = (
-    adhConfig : AdhConfig.IService,
-    adhDone,
-    adhHttp : AdhHttp.Service<any>,
-    adhPreliminaryNames : AdhPreliminaryNames.Service,
-    adhCredentials : AdhCredentials.Service
-) => {
-    return {
-        restrict: "E",
-        template: "<adh-comment-listing data-ng-if=\"display\" data-path=\"{{commentablePath}}\"></adh-comment-listing>"
-                  + "<div data-ng-if=\"display === false\">{{ 'TR__COMMENT_EMPTY_TEXT' | translate }}</div>",
-        scope: {
-            poolPath: "@",
-            key: "@"
-        },
-        link: (scope) => {
-            scope.display = undefined;
-            var commentablePath = scope.poolPath + scope.key + "/";
-
-            var setScope = (path) => {
-                scope.display = true;
-                scope.commentablePath = path;
-            };
-
-            // create commentable if it doesn't exist yet
-            // REFACT: Add Filter "name": scope.key - this requires name index to be enabled in the backend
-            adhHttp.get(scope.poolPath, {
-                elements: "paths",
-                "content_type": RIExternalResource.content_type
-            }).then(
-                (result) => {
-                    if (_.includes(result.data[SIPool.nick].elements, commentablePath)) {
-                        setScope(commentablePath);
-                    } else {
-                        var unwatch = scope.$watch(() => adhCredentials.loggedIn, (loggedIn) => {
-                            if (loggedIn) {
-                                var externalResource = new RIExternalResource({preliminaryNames: adhPreliminaryNames, name: scope.key});
-                                return adhHttp.post(scope.poolPath, externalResource).then((obj) => {
-                                    if (obj.path !== commentablePath) {
-                                        console.log("Created object has wrong path (internal error)");
-                                    }
-                                })["finally"](() => {
-                                    // If the post didn't succeed, somebody else will probably already
-                                    // have posted the resource. The error can thus be ignored.
-                                    setScope(commentablePath);
-                                    unwatch();
-                                });
-                            } else {
-                                scope.display = false;
-                            }
-                        });
-                    }
-                },
-                (msg) => {
-                    console.log("Could not query given postPool");
-                }
-            ).then(adhDone);
-        }
-    };
-};
```

Comment:
This directive is actually used for acceptance tests and it is embedded on policycompass. So it is *not* dead code.